telemetry/jetbrains/src/main/kotlin/software/aws/toolkits/telemetry/generator/TelemetryGenerator.kt

```diff
@@ -40,13 +40,38 @@ fun generateTelemetryFromFiles(
     outputFolder: File
 ) {
     val telemetry = TelemetryParser.parseFiles(defaultDefinitions, inputFiles)
+    val commonMetadataTypes = setOf(
+        "awsAccount",
+        "awsRegion",
```

Comment:
```suggestion
```
`awsAccount` and `awsRegion` are special-case and hydrated through `MetricEventMetadata`, so these should not be included

Reply:
okay